MANIFEST.in

```
@@ -4,19 +4,32 @@ include SYMLINK_CACHE.json
 include requirements.txt
 include shippable.yml
 include bin/ansible-test
+recursive-include docs *
+exclude docs/docsite/rst_warnings
+recursive-exclude docs/docsite/_build *
+recursive-exclude docs/docsite/extensions *.pyc *.pyo
```

This path doesn't exist. Did you mean this?

```suggestion
recursive-exclude docs/docsite/_extensions *.pyc *.pyo
```

---

Fixed